src/ReactCrop.tsx

```diff
@@ -596,6 +608,88 @@ class ReactCrop extends PureComponent<ReactCropProps, ReactCropState> {
     }
   };
 
+  onHandlerKeyDown = (
+    e: React.KeyboardEvent<HTMLDivElement>,
+    handle: 'nw' | 'n' | 'ne' | 'e' | 'se' | 's' | 'sw' | 'w'
+  ) => {
+    const { crop, disabled, minWidth = 0, maxWidth, minHeight = 0, maxHeight, onChange } = this.props;
+    const { width, height } = this.mediaDimensions;
+
+    // Keep the event from bubbling up to the container
+    if (e.key === 'ArrowUp' || e.key === 'ArrowDown' || e.key === 'ArrowLeft' || e.key === 'ArrowRight') {
+      e.stopPropagation();
```

Comment:
Stopping propagation is generally bad practice, is there a reason for it (vs just `preventDefault()`)?

Reply:
I'm trying to change as little as possible in the library and the handlers are currently nested inside of the container. This means that if I click the left arrow when on one of the drag handlers, that if I don't stop the propagation it will propagate up to the container and trigger a move of the entire crop instead of the resize that I want.